### ISSUE TITLE
Typo in config file Config/CarryWeight.cfg

### ISSUE THREAD

**Issue by amelia-ketel** [ServUO/ServUO#5033]
https://github.com/ServUO/ServUO/blob/03202700533adec842459f377f50489cc89786b0/Config/CarryWeight.cfg#L2

Typo in config option.

Backback -> Backpack

**Comment by Gortomo** [comment]
I do not think this is a typo.  If you change it to PlayerBackPack and save the change, the server calls out the error whenever it is checked and uses a default value of Playerbackback.
